python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -60,6 +60,7 @@ def asset(
     key_prefix: Optional[CoercibleToAssetKeyPrefix] = None,
     ins: Optional[Mapping[str, AssetIn]] = ...,
     non_argument_deps: Optional[Union[Set[AssetKey], Set[str]]] = ...,
+    upstream_assets: Optional[Set[CoercibleToAssetKey]] = ...,
```

Comment:
Discussion: What should the new parameter be named? 
Some options:
* `upstream_assets`
* `deps`

Reply:
Some options we've discussed:
* `upstream_assets`
* `deps`

Reply:
* `materialize_after`

Reply:
* `depends_on`